Source/CBLRemoteRequest.m

```diff
@@ -294,8 +294,11 @@ - (void)connection:(NSURLConnection *)connection
             if (cred) {
                 LogTo(RemoteRequest, @"    challenge: useCredential: %@", cred);
                 [sender useCredential: cred forAuthenticationChallenge:challenge];
-                // Update my authorizer so my owner (the replicator) can pick it up when I'm done
-                _authorizer = [[CBLBasicAuthorizer alloc] initWithCredential: cred];
+
+                if ($equal(authMethod, NSURLAuthenticationMethodHTTPBasic)) {
```

Comment:
Why this test? It shouldn't matter here whether basic or digest auth is used; this code is just handling looking up a credential from the keychain.